### PR TITLE
docs: Revise language SDK ecosystem content

### PR DESCRIPTION
**Pull request by charlieegan3**
In the OPA ecosystem, there are a number of language SDKs for different general purpose languages, mostly integrating with the REST API. This PR highlights these at the start of the OPA ecosystem page, adds some other projects for completeness and clarifies the owner where more than one SDK for a language exists.

https://github.com/open-policy-agent/opa/assets/1774239/3f23b442-9228-4c09-af16-6182045237fe

### CODE REVIEW

File: docs/website/content/organizations/turall.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+link: https://github.com/Turall
+title: Tural
```
Comment (anderseknert) [inline docs/website/content/organizations/turall.md:3]: Turall ? :)

**Review by anderseknert — APPROVED** [review]
A typo, I think, but otherwise LGTM

File: docs/website/content/organizations/turall.md (hunk 1 above)
Comment (charlieegan3) [reply docs/website/content/organizations/turall.md:3]: Hmm, on https://github.com/Turall it's Tural.

Comment (charlieegan3) [reply docs/website/content/organizations/turall.md:3]: but perhaps best to use the username

Comment (charlieegan3) [reply docs/website/content/organizations/turall.md:3]: but on https://github.com/open-policy-agent/opa/pull/6811/files#diff-45f53c4dd58caae0eef2f5e763d68be001efd052297d3bc7692e7692c4271017R3 I have used the GH account name.

Comment (anderseknert) [reply docs/website/content/organizations/turall.md:3]: Heh, either is fine

**Pull request merged**